pages/manage/productions/index.test.js

```diff
@@ -0,0 +1,58 @@
+import Index from './Index';
+import Vue from 'vue';
+
+import { mountWithEnvironment } from '../../../test/mountWithEnvironment';
+
+test('returns full list when entering the page', async () => {
+  const wrapper = mountWithEnvironment(Index);
+
+  await Vue.nextTick();
+
+  const listItems = wrapper.findAll('.productions .list-group-item');
+  expect(listItems).toHaveLength(2);
```

Comment:
Could we do an assertion on the expected title(s) of the productions instead, since we mock them anyway? To avoid having a dom selector that could change?

Reply:
An example from vue test utils documentation: https://vue-test-utils.vuejs.org/guides/#getting-started

`expect(wrapper.text()).toContain('Hello world')`